components/Dialogs/src/MDCAlertController.h

```diff
@@ -85,6 +85,23 @@
 + (nonnull instancetype)alertControllerWithTitle:(nullable NSString *)title
                                          message:(nullable NSString *)message;
 
+/**
+ Convenience constructor to create and return a view controller for displaying an alert to the user.
+
+ After creating the alert controller, add actions to the controller by calling -addAction.
+
+ @note Most alerts don't need titles. Use only for high-risk situations.
+ 
+ @note This method receives an @c NSAttributedString for the display message. Use
```

Comment:
```suggestion
 @note Most alerts don't need titles. Use only for high-risk situations.

 @note This method receives an @c NSAttributedString for the display message. Use
```

Reply:
Removing more trailing whitespace.

Reply:
I recommend turning on "Automatically trim trailing whitespace" in the Xcode config:

<img width="912" alt="Screen Shot 2020-03-20 at 9 17 35 AM" src="https://user-images.githubusercontent.com/45670/77167076-b0f3fd80-6a8b-11ea-892c-6f505d61176d.png">

<img width="379" alt="Screen Shot 2020-03-20 at 9 17 32 AM" src="https://user-images.githubusercontent.com/45670/77167082-b3565780-6a8b-11ea-8d3f-52829e03cd05.png">

Reply:
Sweet! Will do! Thanks for the tip!!